src/Files.App/Views/LayoutModes/ColumnViewBase.xaml.cs

```diff
@@ -237,6 +237,9 @@ private void HandleRightClick(object sender, RightTappedRoutedEventArgs e)
 
 		private void FileList_PreviewKeyUp(object sender, KeyRoutedEventArgs e)
 		{
+			if (IsRenamingItem)
```

Comment:
```suggestion
			if (IsRenamingItem || !(e.Key is VirtualKey.Up or VirtualKey.Down or VirtualKey.Right))
```
Although not directly related to the pointed bug, this process is to open a folder when navigating with cursor keys, so it should be disabled when an unrelated key is pressed.